libs/gpu-dawn/sdk.zig

```diff
@@ -69,7 +69,7 @@ pub fn Sdk(comptime deps: anytype) type {
         }
 
         fn linkFromSource(b: *Builder, step: *std.build.LibExeObjStep, options: Options) !void {
-            // branch: generated-2022-08-06
+            // branch: generated-2023-01-12.1673546526
             try ensureGitRepoCloned(b.allocator, "https://github.com/hexops/dawn", "0b704c4acae154ec8d4be7615d18a489f270f6c0", sdkPath("/libs/dawn"));
```

Comment:
@slimsag make Wrench update the hash here `0b704c4acae154ec8d4be7615d18a489f270f6c0` properly